src/test/util_string_tests.cpp

```diff
@@ -58,9 +80,9 @@ BOOST_AUTO_TEST_CASE(ConstevalFormatString_NumSpec)
     // The `*` specifier behavior is unsupported and can lead to runtime
     // errors when used in a ConstevalFormatString. Please refer to the
     // note in the ConstevalFormatString docs.
-    PassFmt<1>("%*c");
-    PassFmt<2>("%2$*3$d");
-    PassFmt<1>("%.*f");
+    PassFmtIncorrect<1, 2>("%*c");
```

Comment:
In commit "test: Prove+document ConstevalFormatString/tinyformat parity" (a911efc6e868ab1a23d05ee8ab3e94f7ecb89e2b)

Maybe drop WrongNumArgs and just pass CorrectArgs? Unless I'm missing something it seems like PassFmtIncorrect could choose the wrong number of args arbitrarily like PassFmt does.

Reply:
The known cases where I was using `PassFmtIncorrect` have now become valid `PassFmt` thanks to your #31174. `PassFmtIncorrect` is no more.